docs/en/ingest-management/fleet/fleet-server.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[fleet-server]]
+= {fleet-server}
+
+beta[]
+
+{fleet-server} is a component used to centrally manage {agent}s. It serves as a
+control plane for updating agent policies, collecting status information, and
+coordinating actions across agents.
+
+//QUESTION: I think it's worth breaking down in detail how data flows between
+// the agent, Fleet Server, Fleet, and the stack, but I might have some details
+// wrong here. Is this correct?
+
+{fleet-server} is the mechanism that connects {agent}s to {fleet} and handles
+all communication between an {agent}, {fleet}, {es}, and {kib}. {fleet-server}:
+
+* Receives status information from connected {agent}s
+* Retrieves agent policies from {fleet} and stores them in {es}
+* Ships agent policies to all connected {agent}s
+* Handles communication with {es} and {kib}
+
```

Comment:
about items 2 and 4... I'm unsure on the wording here - I think for 2:
 it isn't quite accurate, FS doesn't do the storing in ES.

for point 4 - FS doesn't handle 'all communication with Kibana' - FS doesn't actually communicate with Kibana at all.  FS pulls policy info from ES and pushes it to Agents deployed to it.

Reply:
Good points. There were some (possibly outdated) "specs" that suggested Fleet Server did more.  @ruflin Can you provide a breakdown of the communication between Fleet Server and other parts of the stack?

Reply:
Added some notes above. This might also help:

![image](https://user-images.githubusercontent.com/244900/116592073-dc4c4b80-a91f-11eb-92d3-8847672d0aa6.png)

Reply:
Cool. This doesn't look at all the diagram in the design docs. :-)